plugin/src/main/java/com/denizenscript/denizen/events/block/SpongeAbsorbsScriptEvent.java

```diff
@@ -0,0 +1,59 @@
+package com.denizenscript.denizen.events.block;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.block.BlockState;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.SpongeAbsorbEvent;
+
+import java.util.stream.Collectors;
+
+public class SpongeAbsorbsScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // sponge absorbs
+    //
+    // @Location true
+    //
+    // @Group Block
+    //
+    // @Warning this event may in some cases double-fire, requiring usage of the 'ratelimit' command (like 'ratelimit <context.location> 1t') to prevent doubling actions.
```

Comment:
Is this warning correct, or was it copypasted?

If it's correct, uh... why does that happen? 0.o

Reply:
The docs for the SpongeAbsorbEvent mention:
"As this is a physics based event it may be called multiple times for "the same" changes.".
This was true when i was testing so i added a warning.